lib/yao/resources/action.rb

```diff
@@ -0,0 +1,16 @@
+module Yao::Resources
+  module Action
+    def action(id, query)
+      res = POST(action_path(id)) do |req|
+        req.body = query.to_json
+        req.headers['Content-Type'] = 'application/json'
```

Comment:
I think yao's client automaticallt adds these request headers.
https://github.com/yaocloud/yao/blob/master/lib/yao/client.rb#L48
Is it required?